pkgs/desktops/gnome-3/games/aisleriot/default.nix

```diff
@@ -26,6 +34,17 @@ stdenv.mkDerivation rec {
     };
   };
 
+  prePatch = ''
+    patchShebangs cards/meson_svgz.sh
+    patchShebangs data/meson_desktopfile.py
+    patchShebangs data/icons/meson_updateiconcache.py
+    patchShebangs src/lib/meson_compileschemas.py
+  '';
+
+  mesonFlags = [
+    "-Dtheme_kde=false"
+  ];
```

Comment:
These should probably be above `passthru`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.